tools.js

```diff
@@ -59,6 +59,9 @@ module.exports = (function () {
 		timid: {name:"Timid", plus:'spe', minus:'atk'}
 	};
 
+	var init = typeof global.Symbol === 'function' ? Symbol('init') : '_init';
```

Comment:
Sure you haven't, but the modding api should be as friendly as possible imo.
Anyway, I will make them ``_ownInit`` and ``_ownInherit`` when I get proper Git + Internet access.

We can always revisit ES6 features later anyway...